vulnerabilities/importer.py

```diff
@@ -170,6 +183,15 @@ def get_fixed_purl(self):
             raise ValueError(f"Affected Package {self.package!r} does not have a fixed version")
         return update_purl_version(purl=self.package, version=str(self.fixed_version))
 
+    def __lt__(self, other):
+        if not isinstance(other, AffectedPackage):
+            return NotImplemented
+        return self._cmp_key() < other._cmp_key()
+
+    # TODO: Add cache
+    def _cmp_key(self):
+        return (str(self.package), str(self.affected_version_range), str(self.fixed_version))
```

Comment:
If there is no fixed_version or no affected_version_range, their value would be `None` and the string value `"None"` which is not what we want IMHO, but rather `""`
```suggestion
        return (str(self.package), str(self.affected_version_range or ""), str(self.fixed_version or ""))
```